### PR TITLE
Events - removed dummy data

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
1. Removed dummy data
2. Added extra checks for managing the empty event data

Signed-off-by: Rakesh PR <rakesh.pr@mayadata.io>

### CODE REVIEW

File: website/src/resources/events.json [hunk 1]
```diff
@@ -1,31 +1 @@
-[
-    {
```
Comment (Pallavi-PH) [inline website/src/resources/events.json:2]: Can we add event format as a comment?

Comment (IsAmrish) [reply website/src/resources/events.json:2]: or may be adding date format in `interface` will help??

Comment (rakeshPRaghu) [reply website/src/resources/events.json:2]: >or may be adding date format in interface will help??

string will be good enough right! i have no idea on date interface, if you can give me some idea...

Comment (IsAmrish) [reply website/src/resources/events.json:2]: `Date` type

Comment (IsAmrish) [reply website/src/resources/events.json:2]: We need to use DateTime reviver in JSON.parse to be able to use `Date` in the interface.  Anyway, we can go with the string also as long as the string format is consistent . Approving this PR.

**Review by IsAmrish — APPROVED** [review]
LGTM

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by Pallavi-PH — APPROVED** [review]
Lgtm

**Pull request merged**